examples/index.html

```diff
@@ -50,237 +151,256 @@ <h3>Directly access to <span class="text-normal text-italic">bpmn-visualization<
+                                <div class="card-subtitle"></div>
+                            </div>
+                            <div class="card-image">
+                                <img src="static/img/preview_hacktoberfest_diagram.png" class="img-responsive" alt="hacktoberfest themes">
+                            </div>
+                            <div class="card-body">Special Hacktoberfest diagram with Hacktoberfest colors/themes.</div>
+                            <div class="card-footer"></div>
                         </div>
-                    </div>
+                    </a>
                 </div>
             </div>
         </section>
 
         <section class="col-12 mt-2 mb-2">
-            <h2>Miscellaneous</h2>
+            <a id="miscellaneous"  class="anchor-from-summary"></a>
+            <h2 class="bg-dark">Miscellaneous</h2>
             <div class="columns">
                 <div class="col-4 p-2">
-                    <div class="card bg-dark p-2">
-                        <div class="card-image">
-                            <img src="static/img/preview/misc/compare_with_bpmn-js.png" class="img-responsive" alt="compare with bpmn-js">
-                        </div>
-                        <div class="card-header">
-                            <div class="card-title h5">Compare with bpmn-js</div>
-                            <div class="card-subtitle text-gray">compare the libraries on BPMN elements rendering, navigation and API usage</div>
-                        </div>
-                        <div class="card-body"></div>
-                        <div class="card-footer">
-                            <a href="./misc/compare-with-bpmn-js/index.html" target="_blank" class="btn btn-primary">go to preview</a>
+                    <a href="./misc/compare-with-bpmn-js/index.html" target="_blank" class="card-link">
+                        <div class="card p-2">
+                            <div class="card-header">
+                                <div class="card-title">Compare with bpmn-js</div>
+                                <div class="card-subtitle"></div>
+                            </div>
+                            <div class="card-image">
+                                <img src="static/img/preview/misc/compare_with_bpmn-js.png" class="img-responsive" alt="compare with bpmn-js">
+                            </div>
+                            <div class="card-body">Compare the libraries on BPMN elements rendering, navigation and API usage.</div>
+                            <div class="card-footer"></div>
                         </div>
-                    </div>
+                    </a>
                 </div>
             </div>
         </section>
     </div>
+
 </section>
+    <!-- inspired by https://moderncss.dev/pure-css-smooth-scroll-back-to-top/ -->
+    <div class="back-to-top-wrapper float-right">
+        <a href="#top" class="back-to-top-link" aria-label="Scroll to Top">🔝</a>
```

Comment:
we need to adjust it for Chrome display

Reply:
Same not nice icon on Chrome 87 Ubuntu 18

Reply:
is it ok like this? 
icon comeas from spectre which we already use
![image](https://user-images.githubusercontent.com/45601541/100363208-b501f300-2ffc-11eb-862e-1c7f4f22e073.png)

other available icons:
https://picturepan2.github.io/spectre/elements/icons.html